RTS_Data/FormattedData/SIIP/user_descriptors.yaml

```diff
@@ -70,10 +70,21 @@ generator:
 - {custom_name: Output_pct_2, name: output_point_2, unit_system: device_base}
 - {custom_name: Output_pct_3, name: output_point_3, unit_system: device_base}
 - {custom_name: Output_pct_4, name: output_point_4, unit_system: device_base}
-- {custom_name: Base MVA, name: base_mva}
+- {custom_name: MATPOWER BaseMVA, name: base_mva} #TODO just for testing
```

Comment:
remove this line

Reply:
solved via  33fb51f

Reply:
The previous line `- {custom_name: Base MVA, name: base_mva}` needs to be added back.